app/views/organizations/_sidebar.html.erb

```diff
@@ -46,7 +46,7 @@
     <div class="crayons-card crayons-card--secondary p-4">
       <div class="flex items-center mb-4">
         <%= crayons_icon_tag(:post, class: "mr-3 color-base", title: t("views.organizations.side.post.icon")) %>
-        <%= t "views.organizations.side.post.text", count: @stories.size %>
+        <%= t "views.organizations.side.post.text", count: @organization.articles.size %>
```

Comment:
So, since `stories_controller#handle_organization_index` sets up `@stories` to be paginated, we either need to use kaminari's `total_count` or call `size` on something that hasn't been paginated.

I think `total_count` would be a better fit here, since we'd preserve any additional scoping being set in the controller. If we _really_ wanted to count all the organization's articles, then we might rather use the cached count:

```suggestion
        <%= t "views.organizations.side.post.text", count: @organization.articles_count %>
```

Reply:
This wouldn't work, as pointed out by @benhalpern:
> Needs to only show published

because `articles_count` counts unpublished articles too. Instead I actually opted for making a delegated method inside of organizations (added specs for that too).

Reply:
Yes, I also realized that. Added a function that returns the amount of public articles, as to be able to give it a more specific test. That *maybe* adds some unnecessary complexity, take a look at it.